using Exceptions;
namespace ExceptionCatcher
{
    class MainArrayCatch
    {
       static void Main ()
        {
            TryCatchExceptions.Catcher();
            Console.Read();
        }
    }
}
